android/src/main/java/com/sailthru/mobile/rnsdk/JsonConverter.kt

```diff
@@ -0,0 +1,126 @@
+package com.sailthru.mobile.rnsdk
+
+import androidx.annotation.VisibleForTesting
+import com.facebook.react.bridge.*
+import org.json.JSONArray
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Class for handling conversion between Native Maps and Arrays and JSON Objects and Arrays.
+ */
+internal class JsonConverter {
+
+    /**
+     * Convert JSONObject to WritableMap
+     */
+    @Throws(JSONException::class)
+    fun convertJsonToMap(jsonObject: JSONObject): WritableMap {
+        val writableMap: WritableMap = createNativeMap()
```

Comment:
I'm curious if specifying the Type here is a stylistic choice? I don't think that kotlin requires this.

Reply:
I think that's left over from the Java -> Kotlin conversion, but you're right it is unnecessary. Will remove 👍